### PR TITLE
301 bootstrap errorsfield styling

### PR DESCRIPTION
**Pull request by zeroasterisk**
https://github.com/vazco/uniforms/issues/301#issuecomment-310292017

bs4
bs3

### CODE REVIEW

File: demo/package.json [hunk 1]
```diff
@@ -13,8 +13,11 @@
     "antd": "2.10.4",
     "babel-polyfill": "6.23.0",
     "graphql": "0.10.1",
+    "indexof": "0.0.1",
```
Comment (radekmie) [inline demo/package.json]: Do not add it. For more info see component/classes#30.

File: demo/package.json [hunk 2]
```diff
@@ -13,8 +13,11 @@
...
     "material-ui": "0.18.2",
+    "message-box": "0.0.2",
     "meteor-node-stubs": "0.2.11",
+    "moment": "^2.18.1",
```
Comment (radekmie) [inline demo/package.json]: Those are dependencies of our dependencies - why you've added them?

File: packages/uniforms-bootstrap4/src/ErrorsField.js [hunk 3]
```diff
@@ -6,7 +6,7 @@ import nothing        from 'uniforms/nothing';
 
 const ErrorsField = ({className, children, ...props}, {uniforms: {error, schema}}) =>
     (!error && !children) ? nothing : (
-        <div className={classnames('card', className)} {...filterDOMProps(props)}>
+        <div className={classnames('card', 'card-outline-danger', 'text-danger', className)} {...filterDOMProps(props)}>
```
Comment (radekmie) [inline packages/uniforms-bootstrap4/src/ErrorsField.js]: How about `'card card-outline-danger text-danger'` instead?

**Comment by zeroasterisk** [comment]
I rebased to merged commits and ignored the package.json, then force pushed.

**Pull request merged**